pkg/cloud/amazon/s3_storage_test.go

```diff
@@ -419,7 +419,8 @@ func TestS3BucketDoesNotExist(t *testing.T) {
 	q.Add(cloud.AuthParam, cloud.AuthParamImplicit)
 	q.Add(S3RegionParam, "us-east-1")
 
-	bucket := "invalid-bucket"
+	// Very long, very random bucket name, that hopefully nobody will ever create.
+	bucket := "VIBK1H88MOJ665V2RAPVH6X3TWUS0HCWTW5A27AFPPLHMABKH7X445K86K1BP2"
```

Comment:
I know this is what we merged to master, but I question whether this is what we want.  Can we instead generate a UUID or something?

Reply:
why does it matter?